laurent22/joplin
# Mobile: Setting to disable spellcheck in beta editor

# Summary
 * Adds a setting that allows users to disable automatic spellcheck in the beta editor.
 * This setting defaults to `true`.

![](https://user-images.githubusercontent.com/46334387/186555819-7e3a592e-a5e8-430b-a165-b8bcba40d745.png)

# Testing plan
0. Disable the beta editor. The setting should be invisible.
1. Enable the beta editor. The new setting should be visible.
2. Disable automatic spellcheck.
3. Edit a note with the beta editor, misspell something.
4. Enable automatic spellcheck.
5. Edit the same note with the beta editor.

# Notes
 * Red underlines don't appear on iOS. Tapping on a misspelled word still shows suggestions.
